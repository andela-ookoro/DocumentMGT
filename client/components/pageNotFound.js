import React from 'react';

const pageNotFound = () => (
  <div className="container">
    <div className="welcomeDiv">
      <img src="/logo.png" id="logoImg" alt="logo" />
      <span className="logoName" id="brandName"> Doc Hub </span>
      welcome .....
    </div>
    <div className="row">
      <div className="col s12">
        <h2> Sorry Page this page was not found </h2>
        <h2>
          Please, visit our
          <a href="#/" id="lnkHome"> home page </a> for more information
        </h2>
      </div>
    </div>
  </div>
);

export default pageNotFound;
