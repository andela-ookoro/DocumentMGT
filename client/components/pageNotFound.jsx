import React from 'react';

const pageNotFound = () => (
  <div className="container" >
    <div className="row">
      <div className="col s12">
        <h5> <i className="large material-icons">visibility_off</i></h5>
        <h5> Sorry Page this page was not found </h5>
        <h6>
          Please, visit our
          <a href="#/" id="lnkHome"> home page </a> for more information
        </h6>
      </div>
    </div>
  </div>
);

export default pageNotFound;
