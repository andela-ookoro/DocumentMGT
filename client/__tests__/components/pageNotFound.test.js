import React from 'react';
import Renderer from 'react-test-renderer';
import pageNotFound from '../../components/pageNotFound';

describe('rendering', () => {
  it('should render content as describe in the component', () => {
    const component = Renderer.create(
      <pageNotFound />,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});