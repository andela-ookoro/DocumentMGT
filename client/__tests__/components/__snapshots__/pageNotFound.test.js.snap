// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pageNotFound component should render content as describe in the component 1`] = `<pageNotFound />`;
